cmd/user-agent/src/pages/DIDManagement.vue

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+<template>
+    <div class="content">
+        <div class="md-layout">
+            <div
+                    class="md-layout-item md-medium-size-100 md-xsmall-size-100 md-size-100"
+            >
+                <md-card class="md-card-plain">
+                    <md-card-header data-background-color="green">
+                        <h4 class="title">Create your DID</h4>
+                        <p class="category"> Create button will create and save the did in DID store</p>
+                    </md-card-header>
+                    <md-card-content style="background-color: white;">
+                        <md-field>
+                        </md-field>
+                        <select id="selectVC" v-model="selectType" style="color: grey; width: 200px; height: 35px;">
+                            <option value="" disabled="disabled">Select Key Type</option>
+                            <option value="Ed25519">Ed25519VerificationKey2018</option>
```

Comment:
change from Ed25519VerificationKey2018 to Ed25519

Reply:
done

Reply:
@fqutishat why the change?

Reply:
Ok - saw the model. It has both `KeyType` and `Type`. Why both? @sandrask asked the same question [here](https://github.com/trustbloc/edge-agent/pull/105#pullrequestreview-398296422)

Reply:
Key Type is the crpyto we support .
Type is the signature type.  Example : 
  {
                        "id":"key-recovery",
                        "type":"JwsVerificationKey2020",
                        "value":recoveryKeyset.signaturePublicKey,
                        "encoding":"Jwk",
                        "keyType":Ed25519,
                        "recovery":true
                    }

Reply:
Ah, so i guess keytype exists solely because of jsonwebsignature 2020. Thanks